tests/remove-spaces.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const assert = require('assert');
+const expect = require('chai').expect;
+const HTMLBarsMinifier = require('../lib/htmlbars-minifier');
+
+const minifier = new HTMLBarsMinifier('foo', {
+	removeSpacesAroundTags: true
+});
+
+describe('removeSpacesAroundTags', () => {
+	it('should escape special chars', () => {
```

Comment:
I don't quite understand this test case, you are testing removing spaces, why would you mention special chars now? Also, the test input does not have any space to remove whatsoever.